### ISSUE TITLE
Scheduler: Periodcally schedul jobs

### ISSUE THREAD

**Issue by WolfgangEibner** [backmeup/backmeup-prototype#56]
- Reschedule periodical jobs after executing them
- Should periodical jobs get scheduled until they are deleted or do we have same active/inactive flag?
- Should we stop rescheduling a job if it has an error? If so, how can the user continue the job?

**Comment by WolfgangEibner** [comment]
Rescheduling should be done in f8c704cc3a8435cea9a451ebe074f5ddc4208ad3.

**Comment by rsimon** [comment]
Yes - rescheduling happens; are currently re-scheduled until deleted by the user.

**Comment by WolfgangEibner** [comment]
We should discuss the two remaining points at the next meeting.

**Comment by WolfgangEibner** [comment]
According to today's meeting:
- Stop rescheduling a job if it gets deleted
- Introduce an onHold flag to (de-)activate a job, including according changes in REST API (onHold status, feature to put job onHold or reactivate it)
- Put a job onHold after it ended with an error
- UI should also offer a possibility to deactivate/reactivate (pause/continue) a job

**Comment by rsimon** [comment]
Committed fix: after a Job is deleted from the DB, it is no longer rescheduled

**Comment by WolfgangEibner** [comment]
onHold Flag introduced in 7a42104079f45796dedc39477b00a39ec29582cb.

Still needed:
- changes in REST API (onHold status, feature to put job onHold or reactivate it)
- Put a (rescheduling) job onHold after it ended with an error (do we want that)?
- UI should also offer a possibility to deactivate/reactivate (pause/continue) a job

**Comment by keros** [comment]
Added isOnHold status to the REST GET interface (/jobs/username -> getjobs)
